prepare_benchmark

```diff
@@ -43,7 +48,7 @@ if __name__ == '__main__':
         'Which benchmark to run.'
     ))
 
-    scale_factor_required = any(x in sys.argv for x in ('tpcds', 'tpch', 'ssb'))
+    scale_factor_required = any(x in ''.join(sys.argv) for x in ('tpcds', 'tpch', 'ssb'))
```

Comment:
That should not be required.

Reply:
I tried the script locally with python3.6 and the "scale_factor_required" was always False. that is why I make this change.

Reply:
This is strange. I would like to understand why.

However, if you want to keep it like this, then you can remove the `any`.

Reply:
I think we need the `any` in order to loop around the benchmark names, right?